cmd/queue/main.go

```diff
@@ -200,7 +201,22 @@ func (h *healthServer) quitHandler(w http.ResponseWriter, r *http.Request) {
 	// no guarantee that container termination is done only after
 	// removal from service is effective, but this has been showed to
 	// alleviate the issue.
-	time.Sleep(quitSleepSecs * time.Second)
+	timer := time.NewTimer(quitSleepSecs * time.Second).C
+	ticker := time.NewTicker(time.Second).C
+
+CheckAllDone:
+	for {
+		select {
+		case <-timer:
```

Comment:
I believe you can just use `<-time.After(quitSleepSecs * time.Second)`

Reply:
This is a nice code, thanks.